packages/shell-chrome/assets/panel.html

```diff
@@ -26,238 +26,273 @@
             border-right: 4px solid transparent;
             border-top: 6px solid #444;
         }
+
+        [x-cloak]{
+            display: none;
+        }
     </style>
 </head>
 
-<body class="antialiased">
+<body class="loading antialiased">
 
-    <div class="bg-white flex flex-col relative h-screen w-full mx-auto">
-        <div 
-        id="header"
-        x-data="{
-            version: null,
-            latest: null,
+    <div 
+    id="app" 
+    x-cloak
```

Comment:
Could we make it so that we can display a loading state before Alpine.js kicks in?

So maybe just having a `<div>Devtools loading...</div>`

(currently we're x-cloak-ing everything and due to the way we're loading the CSS we have no styling before the JS loads)

Reply:
This is what I came up with, not particularly beautiful but it'll do until we improve the CSS situation

```html
        .preload {
            display: flex;
            align-items: center;
            justify-content: center;
            font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
            padding: 20px;
        }
</style>
<body class="loading antialiased">

+    <div x-data x-show="false" class="preload">Devtools loading...</div>

    <div
    id="app"
    x-cloak
```

Reply:
I've merged the Tailwind fix in, so maybe we can use Tailwind classes (I've put the CSS load in the `<head>` so I think it's render-blocking

Reply:
Eg. this is using the latest master, there's basically no jump

![Kapture 2020-11-23 at 20 15 15](https://user-images.githubusercontent.com/6459679/100010943-f0b47700-2dc8-11eb-8009-b944f1fc75d1.gif)

Reply:
Yeah good shout, I did nearly separate out the CSS to get to that first paint quicker. This works for sure. My shout would be to knock it back a touch with some grey and take care of any body margin/padding defaults:

```
body{
    margin: 0;
    padding: 0;
}
.preload {
    display: flex;
    align-items: center;
    justify-content: center;
    height: 100vh;
    color: #888;
    background-color: #eee;
    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
    padding: 20px;
}
```

Reply:
Feel free to axe that `loading` class that I've left on the body too...not sure what I was doing there.

Reply:
Alpine kicks in pretty quick now so I've not added that preload in the end, resolved the conflicts for you though 😄